docs/cosmos-hub/gaiacli.md

```diff
@@ -219,19 +218,17 @@ You can simulate a transaction without actually broadcasting it by appending the
 `--dry-run` flag to the command line:
 
 ```bash
-gaiacli tx send <destination_cosmosaccaddr> 10faucetToken \
+gaiacli tx send <sender_key_name_or_address> <destination_cosmosaccaddr> 10faucetToken \
   --chain-id=<chain_id> \
-  --from=<key_name> \
   --dry-run
 ```
 
 Furthermore, you can build a transaction and print its JSON format to STDOUT by
 appending `--generate-only` to the list of the command line arguments:
 
 ```bash
-gaiacli tx send <destination_cosmosaccaddr> 10faucetToken \
+gaiacli tx send <sender_address> <destination_cosmosaccaddr> 10faucetToken \
```

Comment:
```suggestion
gaiacli tx send <sender_key_name_or_address> <recipient_address> 10faucetToken \
```

Reply:
Hold on, that was left as-is cause it's the `--generate-only` syntax example.